src/nvim/drawline.c

```diff
@@ -2697,6 +2812,10 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, bool nochange,
       if (filler_todo <= 0) {
         need_showbreak = true;
       }
+      if (statuscol.draw) {
+        statuscol.draw = vim_strchr(p_cpo, CPO_NUMCOL) == NULL;
+        statuscol.hl_idx = 0;
+      }
```

Comment:
What's this block for?

Reply:
Thanks for taking the time to review.

@khalidchawtany reported an [issue](https://github.com/neovim/neovim/pull/20621#issuecomment-1337095633) with `cpoptions+=n` set. I'm not sure how that option should interact with `statuscolumn`, see https://github.com/neovim/neovim/pull/20621#issuecomment-1339197048. This block falls back to the default drawing routines for wrapped lines when `cpoptions+=n` is set. This works well enough, but it might not look right if a user changes the sign/fold column location in their `statuscolumn`.

Reply:
Sounds ok. That information definitely belongs in a comment.